tox.ini

```diff
@@ -61,7 +61,7 @@ commands =
 basepython = python3.8
 
 [testenv:lint]
-basepython = python3.8
+basepython = python3
```

Comment:
I see you here too :stuck_out_tongue_winking_eye:

Reply:
ta ta ta